src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/UploadEnvelopeDocumentsService.java

```diff
@@ -88,6 +88,8 @@ private void processEnvelope(CloudBlobContainer blobContainer, Envelope envelope
         CloudBlockBlob blob = null;
         Optional<String> leaseId = Optional.empty();
 
+        log.info("Processing envelope. Container {}, File {}", containerName, zipFileName);
```

Comment:
We already have processing zip file in the logs 
<img width="1377" alt="image" src="https://user-images.githubusercontent.com/13840402/87153905-23ddea00-c2b0-11ea-8725-36e9d6cd7cf4.png">

Reply:
ZipFileProcessor::process which produces this log entry is called from BlobProcessorTask (exactly - from FileContentProcessor) and from UploadEnvelopeDocumentsTask (exactly - from UploadEnvelopeDocumentsService). In the first case there is a log entry from the calling method, in the second case - no, unless this new line is merged. With this new line reading logs and finding exact workflows will be less confusing.

Reply:
should the existing one be deleted then?

Reply:
No. The existing log is in BlobProcessorTask, the method writing this log entry is called by 2 callers, one of these callers writes a log entry, the other caller does not. This PR is fixing this - both callers will write log entries and thus the whole workflow will be traceable.